onnxruntime/wasm/api.cc

```diff
@@ -1,9 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include "api.h"
+#ifdef ENABLE_TRAINING_APIS
```

Comment:
So, the training pipeline will also cover wasm tests, or the webassembly pipelines will also cover training?